docs/source/nbconvert_library.ipynb

```diff
@@ -511,27 +511,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The following demonstration, as requested in [a GitHub issue](https://github.com/ipython/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
+    "The following demonstration, as requested in [a GitHub issue](https://github.com/jupyter/nbconvert/pull/137#issuecomment-18658235), adds the ability to exclude a cell by index. \n",
```

Comment:
Github audit logs say it was deleted by @fperez's account last month:
https://github.com/organizations/ipython/settings/audit-log?utf8=%E2%9C%93&q=action%3Arepo.destroy%20repo%3Aipython%2Fnbconvert

Hopefully Fernando's just been so busy that he forgot this, and we don't have a really weird hacker. ;-)